src/GraphQl/Queries/Queries.ts

```diff
@@ -186,18 +172,8 @@ export const USER_ORGANIZATION_LIST = gql`
 
 // to take the organization event list
 export const ORGANIZATION_EVENT_LIST = gql`
-  query EventsByOrganization(
-    $id: ID!
-    $filterByTitle: String
-    $filterByDescription: String
-  ) {
-    eventsByOrganization(
-      id: $id
-      where: {
-        title_contains: $filterByTitle
-        description_contains: $filterByDescription
-      }
-    ) {
```

Comment:
Why we need to remove the filters ?

Reply:
refer this
https://thepalisadoes-dyb6419.slack.com/archives/C01RPC00K0D/p1674557631837709

Reply:
![image](https://user-images.githubusercontent.com/67755128/216835866-eacb5242-14ec-494a-8d22-658524aee0e2.png)

same thing when this.
in the backend there is no such query mentioned to handle the where argument

Reply:
@noman2002 @VenomFate-619 we need to create `eventsByOrganizationConnection` for filtering out events with `where` parameter here. @palisadoes Should I open an issue for this in `talawa-api`?